common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerValidator.java

```diff
@@ -59,10 +59,11 @@
     private static boolean sShouldTrustDebugBrokers = BuildConfig.DEBUG;
 
     public static void setShouldTrustDebugBrokers(final boolean shouldTrustDebugBrokers) {
-        if (!BuildConfig.DEBUG && shouldTrustDebugBrokers) {
-            Logger.warn(TAG, "You are forcing to trust debug brokers in non-debug builds.");
+        if (BuildConfig.DEBUG || BuildConfig.SHOULD_TRUST_DEBUG_BROKERS) {
+            BrokerValidator.sShouldTrustDebugBrokers = shouldTrustDebugBrokers;
+            return;
         }
-        BrokerValidator.sShouldTrustDebugBrokers = shouldTrustDebugBrokers;
+        throw new RuntimeException("Cannot trust debug brokers in non-debug builds.");
```

Comment:
What if the value supplied to the method is actually `false`? I don't think we need to throw RuntimeException in that case.

Reply:
Thanks for catching this. Fixed in 0e7188f